projects/origin-release/pipeline.yaml

```diff
@@ -31,103 +31,548 @@ parameters:
   description: Git source context base to use for building the Origin release environment image
   value: projects/origin-release
   required: false
+- name: PUBLISH_REPO
+  displayName: Publish Repo
+  description: Container image registry repo to use for pushing the release environment images
+  value: docker.io/openshift
+  required: false
 objects:
 - kind: ImageStream
   apiVersion: v1
   metadata:
     labels:
       app: "${NAME}"
     name: "${NAME}"
+  spec:
+    tags:
+    - from:
+        kind: DockerImage
+        name: centos:7
+      name: x86_64-centos-7
+    - from:
+        kind: DockerImage
+        name: centos/aarch64:7
+      name: aarch64-centos-7
+    - from:
+        kind: DockerImage
+        name: ppc64le/centos:7
+      name: ppc64le-centos-7
```

Comment:
My intention with adding IS tags for the arch-specific images was to allow for those images to be rebuilt when updates are available from upstream. I had to remove the scheduled refreshes, though because I was hitting an issue where all of the builds were getting re-triggered every time the scheduled IS update ran. I'm happy to remove these tags and reference the images directly in the base image builds if that is preferable.